contrib/charts/cert-manager/values.yaml

```diff
@@ -6,29 +6,41 @@ replicaCount: 1
 image:
   repository: quay.io/jetstack/cert-manager-controller
   tag: v0.2.2
-  pullPolicy: Always
+  pullPolicy: IfNotPresent
 
 createCustomResource: true
 
 rbac:
-  enabled: true
+  create: true
+  serviceAccountName: default
+
+# Optional additional arguments
+extraArgs: []
+  # - --cluster-resource-namespace=ingress
```

Comment:
Can we change 'ingress' to 'kube-system'? That is currently our default for this field, and introducing 'new' namespaces could cause confusion 😄

Reply:
This was meant as an example. Wouldn't you only set this if you want a different namespace? Maybe we should add a comment making that clear?